### ISSUE TITLE
 Upcoming Appointment Details: VAVC at Home - Accessibility Issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16829]
## What Happened?
**VoiceOver / TalkBak**

1. With the appropriate screen reader enabled on iOS and Android, the a11yHint for the back button is not being picked up 
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)
2. With the appropriate screen reader enabled on iOS and Android, the "...visit VA.gov" text is incorrectly picked up/pronounced
3. With VoiceOver enabled on iOS, the "Prepare for video visit" hyperlink is not announced as a link

**Axe / Accessibility Scanner**
4. The "Prepare for video visit" hyperlink does not meet minimum 44px by 44px touchpoint requirements

**Voice Control / Voice Access**
5. With Voice Control enabled on iOS, the "Prepare for video visit" hyperlink is not able to be accessed by saying the attached number (the hyperlink has no attached number) or by saying "tap prepare for video visit"

**Keyboard**
6. With Keyboard linked to iOS, the "Prepare for video visit" hyperlink is not being focused and can not be accessed

## Specs:
- Device: Android -Google Pixel 3XL(v.11) with TalkBack and Accessibility Scanner enabled
- Device: iOS - iPhone XR (v.14.2) with VoiceOver and Voice Control enabled
- Bluetooth Keyboard
- User: marcy.nadeau@va.gov

## Steps to Reproduce
**VoiceOver / TalkBak**

For 2:
1. With the appropriate screen reader enabled on iOS or Android, navigate to a VA Video Connect at home page
2. Select the "...visit VA.gov" text in the "Need to cancel" section
3. Note that VA.gov is picked up/pronounced as Vaw [dot] gov on Android and as VA [dot] governor on iOS

For 3: 
1. With VoiceOver enabled on iOS, navigate to a VA Video Connect at home page
2. Select the "Prepare for video visit" hyperlink
3. Note that the screen reader does not announced it as a link 

**Axe / Accessibility Scanner**

For 4:
1. Navigate to a VA Video Connect at home page 
2. Enable Axe or Accessibility Scanner on Android 
3. Note that the "Prepare for a video visit" hyperlink is flagged

**Voice Control / Voice Access**

For 5:
1. With Voice Control enabled on iOS, navigate to a VA Video Connect at home page
2. Note that the "Prepare for a video visit" hyperlink is not able to be accessed by saying an attached number (the hyperlink has no attached number) or by saying "tap prepare for a video visit"

**Keyboard**

For 6:
1. With Keyboard linked to iOS, navigate to a VA Video Connect at home page
2. The back button will be in focus or press tab to place it in focus
3. Press the down arrow key. The add to calendar hyperlink will be in focus
4. Press the down arrow key. The join session button will be in focus
5. Press the down arrow key. The visitVA.gov hyperlink will be in focus (skipping the prepare for video visit hyperlink)

## Desired Behavior
**VoiceOver / TalkBak**

For 2:
- With the appropriate screen reader enabled on iOS or Android, the "...visit VA.gov" text will be picked up/pronounced as VA [dot] gov

For 3:
- With VoiceOver enabled on iOS, the "Prepare for video visit" hyperlink will be announced as a link 

For 4:
- With Axe / Accessibility Scanner enabled on Android, the "Prepare for video visit" hyperlink will meet minimum 44px by 44px touchpoint requirements

For 5:
- Using Voice Control on iOS, the "Prepare for a video visit" hyperlink will be able to be selected by saying the attached number or by saying "tap Prepare for a video visit"

For 6:
- With Keyboard linked to iOS, the "Prepare for a video visit" hyperlink will be able to be placed in focus and selected with the tab or navigational keys

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 3 - Medium

## Linked Story (s)
- Story #15714

## Screen Shot (s) and Additional Information
2. Showing the affected areas 
![VA Appointment - Android Screen Reader.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/8e26abb2-f656-4cb4-bfbd-93783bfc9f19)

4. Axe / Accessibility Scanner Bug 🔽
![VA Video Connect at Home - Android Touchpoint.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/39d27529-5e49-4438-9bf8-4f793411328f)

5. Voice Control iOS Bug 🔽
![VA Video Connect at Home - iOS Voice Control.PNG](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/bf41ac04-11df-4af5-9606-ae245bfac35d)

**Comment by digitalken** [comment]
Testing...
